cf-custom-resources/lib/wkld-cert-validator.js

```diff
@@ -175,7 +175,7 @@ exports.handler = async function (event, context) {
   domainName = props.DomainName;
   rootDNSRole = props.RootDNSRole;
   isCloudFrontCert = props.IsCloudFrontCertificate;
-  certificateDomain = isCloudFrontCert ? `${serviceName}.${envName}.${appName}.${domainName}` : `${serviceName}-nlb.${envName}.${appName}.${domainName}`;
+  certificateDomain = isCloudFrontCert ? props.TruncStaticSiteDefaultDomain : `${serviceName}-nlb.${envName}.${appName}.${domainName}`;
```

Comment:
Oki I feel like it's rather inconsistent that static site is passing a certificate domain, while LBWS is letting the customer resource construct the domain. I think ideally the code-path for both cases should be the same for better maintainability. 

I wonder if ⬇️ makes sense.

In CloudFormation template
```yaml
  CertificateValidatorAction:
    Properties:
      IsCloudFrontCertificate: true
      ServiceDomain: {{ .ServiceDomain }}
      Aliases: {{ if .StaticSiteAlias }} [{{ quote .StaticSiteAlias }}] {{ else }} [] {{ end }}
```

In `cloudformation/stack/static_site.go`
```go
content, err := s.parser.ParseStaticSite(template.WorkloadOpts{
    ServiceDomain: truncateDomainName(s.name, s.env, s.app, s.appInfo.Domain)
}
```

In `cloudformation/stack/lbws.go`
```go
content, err := s.parser.ParseLBWS(template.WorkloadOpts{
    NLBCertDomain: fmt.Sprintf("%s-nlb.%s.%s.%s",s.name, s.env, s.app, s.appInfo.Domain)
```